server.js

```diff
@@ -31,11 +35,88 @@ app.get('/api', function(req, res) {
 
   countryTrends.returnTrendsByCountry(function(results) {
     console.log("results", results);
-    res.status(201);
+    res.status(200);
     res.send(results);
   });
 });
 
+app.get('/db', function(req, res) {
+  Trend.find(function(err, trends) {
+    if (err) {
+      throw err;
+    }
+    console.log('TRENDS', trends);
+    res.status(200);
+    res.send(trends);
+  });
+});
+
+
+
+
+
+// TESTING NEW API CALL
+
+var Twitter = require('twitter');
+
+var client = new Twitter({
+  consumer_key: process.env.CONSUMER_KEY,
+  consumer_secret: process.env.CONSUMER_SECRET,
+  access_token_key: process.env.ACCESS_TOKEN_KEY,
+  access_token_secret: process.env.ACCESS_TOKEN_SECRET
+});
+
+app.get('/test', function(req, res) {
+
+  // specify id (WOEID) for each city then make a get request to the end point.
+  // These are stored on my local DB atm (Simon).
+  // Be careful not to call this same location twice or it will duplicate data in the DB.
+  // Maybe need to figure out how to prevent this.
+
+  // var params = {id: 2487956}; // San Francisco
+  // var params = {id: 2487889}; // San Diego
+  // var params = {id: 2379574}; // Chicago
+  // var params = {id: 2450022}; // Miami
+  // var params = {id: 2459115}; // New York (City)
+  // var params = {id: 2514815}; // Washington (D.C.)
+  // var params = {id: 2490383}; // Seattle
+  // var params = {id: 2436704}; // Las Vegas
+  // var params = {id: 2475687}; // Portland
+  // var params = {id: 2391279}; // Denver
+  // var params = {id: 2486340}; // Sacramento
+  // var params = {id: 2442047}; // Los Angeles
+  // var params = {id: 2471390}; // Phoenix
+  // var params = {id: }; // 
+
+  client.get('trends/place', params, function(error, tweets, response){
+    if (error) {
```

Comment:
@SimonDing87  `params` on line 91 breaks the route since params isn't defined. I'm commenting out line 76 so it won't break the route.